### PR TITLE
Rf rb item detail pretty

### LINKED ISSUE #16: 16. As a user, I want the item detail screen to look pretty so the app looks professional and welcoming.
AC:

* The item detail screen (the screen a user sees when they tap on one of their items in the shopping list) looks something like the following (or equivalent, use your creativity!):

item-detail

### LINKED ISSUE #17: 17. As a user, I want the add an item screen to look pretty so the app looks professional and welcoming.
AC:

* The add an item screen (the screen a user uses to add a new item to their list) looks something like the following (or equivalent, use your creativity!):

add-item

### PR DESCRIPTION
**Pull request by rbompiani**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Re-styled AddItem , ItemDetail , Nav bar and NewList components. Moved Header into Nav with conditional styling for nav items. Created new collection in database to store human-readable list names.

## Related Issue

closes #16 closes #17 

## Acceptance Criteria

The item detail screen (the screen a user sees when they tap on one of their items in the shopping list) looks something like the following (or equivalent, use your creativity!):

The add an item screen (the screen a user uses to add a new item to their list) looks something like the following (or equivalent, use your creativity!):

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

NewList - Before
AddItem - Before
ItemDetail - Before
Nav-Before

### After

NewList - After
AddItem - After
ItemDetail - After

## Testing Steps / QA Criteria

When creating a new list, check the database to confirm that the list name is being saved to listName collection
Navigate the app, look for the pretty.

### CODE REVIEW

**Review by audracodes — APPROVED** [review]
Great work! Looks awesome!

**Review by indiragp — APPROVED** [review]
Everything looks so pretty :) Great job!

File: src/components/AddItems.js [hunk 1]
```diff
@@ -65,66 +78,96 @@ const AddItems = () => {
...
+    <Box bg="brand.600">
+      <Center textStyle="roundedCorners">
+        <Text textStyle="h2" orientation="horizonal">
+          Add Your Items!
+        </Text>
+      </Center>
+      <Center bg="white">
+        <form onSubmit={handleSubmit}>
+          <Box my={10}>
+            <FormLabel textAlign="center" fontSize="20px" htmlFor="item-name">
```
Comment (indiragp) [inline src/components/AddItems.js:90]: There is a missmatch between htmlFor (item-name) and the  name of the input (itemName) making an accessibility tester think that there is no label for the input.

Comment (rachel-fischoff) [reply src/components/AddItems.js:90]: Thanks! We had to wrap the input in the form :)

**Pull request merged**